static/js/publisher-pages/pages/Settings/SettingsForm.tsx

```diff
@@ -0,0 +1,515 @@
+                    currentValues={settingsData?.blacklist_countries}
+                    register={register}
+                    setValue={setValue}
+                    getValues={getValues}
+                    control={control}
+                    disabled={getValues("country_keys_status") === "include"}
+                  />
+                </>
+              )}
+            </Col>
+          </Row>
+
+          <div className="u-fixed-width">
+            <hr />
+          </div>
+
+          <Row className="p-form__group">
+            <Col size={2}>
+              <label
+                className="p-form__label"
+                htmlFor="update_metadata_on_release"
+              >
+                Update metadata on release:
+              </label>
+            </Col>
+            <Col size={8}>
+              <div className="p-form__control">
+                <label className="p-checkbox">
+                  <input
+                    type="checkbox"
+                    className="p-checkbox__input"
+                    {...register("update_metadata_on_release", {
+                      setValueAs: (value) => value === "on" || value === true,
+                    })}
+                  />
+                  <span className="p-checkbox__label">Enabled</span>
+                  <br />
+                  <span className="p-form-help-text">
+                    The information on the Listing page of this snap will be
+                    automatically updated to the version in snapcraft.yaml of
+                    the latest revision pushed to the stable channel. If you
+                    manually edit the Listing page, the automatic updates will
+                    be turned off.{" "}
+                    <a href="/docs/snapcraft-top-level-metadata">Learn more</a>.
+                  </span>
+                </label>
+                {getValues("update_metadata_on_release") && (
+                  <Notification severity="caution" title="Warning">
+                    This snap is set to have its metadata updated when a new
+                    revision is published in the stable channel. Any changes you
+                    make here will be overwritten by the contents of any snap
+                    published. If this is not desirable, please disable “Update
+                    metadata on release” for this snap.
+                  </Notification>
+                )}
+              </div>
+            </Col>
+          </Row>
+
+          <div className="u-fixed-width">
+            <hr />
+          </div>
+
+          <Row className="p-form__group">
+            <Col size={2}>
+              <label className="p-form__label" id="collaboration-label">
+                Collaboration:
+              </label>
+            </Col>
+            <Col size={8}>
+              <div
+                className="p-form__control"
+                aria-labelledby="collaboration-label"
+              >
+                <a
```

Comment:
I don't think it's worth the overhead. It doesn't do anything special in the case of a standard link. If anything it will just add weight to the page which seems unnecessary.